services/web/server/requirements/constraints.txt

```diff
@@ -10,3 +10,7 @@
 #
 # SEE https://github.com/joerick/pyinstrument#v400release
 pyinstrument<4
+
+
+# SEE services/web/server/tests/unit/isolated/test_utils.py::test_yarl_url_compose_changed_with_latest_release
+yarl<1.6
```

Comment:
are we not planning to unpin this one?

Reply:
The test still fails with the latest version. It needs some refactoring to adapt to the latest version but for the moment we do not have enough bandwidth and this is not that important